h2o-algos/src/main/java/hex/glm/GLMModel.java

```diff
@@ -828,6 +829,21 @@ public GLMOutput(GLM glm) {
       String[] cnames = glm._dinfo.coefNames();
       String [] names = glm._dinfo._adaptedFrame._names;
       String [][] domains = glm._dinfo._adaptedFrame.domains();
+      if(glm._parms._family == Family.quasibinomial){
+        double [] mins = glm._dinfo._adaptedFrame.lastVec().mins();
+        double [] maxs = glm._dinfo._adaptedFrame.lastVec().maxs();
+        double l = mins[0];
+        double u = maxs[0];
+        if(!(l<u))
+          throw new IllegalArgumentException("quasibinomail family expects response to have two distinct values");
+        for(int i = 0; i < mins.length; ++i){
+          if((mins[i]-l)*(mins[i]-u) != 0)
+            throw new IllegalArgumentException("quasibinomail family expects response to have two distinct values, got mins = " + Arrays.toString(mins) + ", maxs = " + Arrays.toString(maxs));
+          if((maxs[i]-l)*(maxs[i]-u) != 0)
```

Comment:
I think this check is technically redundant because if mins have only two values l = min and u = max then maxs cannot have different values.